src/common/api_strings.hpp

```diff
@@ -47,6 +48,14 @@
 #define OTBR_ROLE_NAME_ROUTER "router"
 #define OTBR_ROLE_NAME_LEADER "leader"
 
+#define OTBR_DHCP6_PD_STATE_NAME_DISABLED "disabled"
```

Comment:
1. `disabled`, `stopped` and `running` are common states used in OT. Maybe the constants can have more general names.
2. I feel it's better to define them as C++ constants instead of macros. May consider `constexpr const char *`.

Reply:
I was debating where to put them actually, how about src/dbus/common/constants.hpp

Reply:
LGTM for now because we don't have other state constants.